zospy/functions/lde.py

```diff
@@ -71,3 +71,67 @@ def surface_change_type(surface: _ZOSAPI.Editors.LDE.ILDERow, new_type: constant
     # Apply
     new_surface_type_settings = surface.GetSurfaceTypeSettings(new_type)
     surface.ChangeType(new_surface_type_settings)
+
+
+def find_surface_by_comment(
+    lde: _ZOSAPI.Editors.LDE, comment: str, case_sensitive: bool = False
```

Comment:
I would make `case_sensitive` `True` by default, so the function looks for exact matches unless specified otherwise. This also goes for the NCE version.

Reply:
I agree with this and have altered it in `find_surface_by_comment()`

Reply:
Done